packages/gui/src/components/chiatools/logViewer/LogViewer.tsx

```diff
@@ -0,0 +1,624 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+
```

Comment:
While `nodeIntegration` is set to `true` in creating browser window, we usually offload functions, that are not expected to run on web browser, to electron's main process from renderer process for security.

Please see how we let electron main process handle local files and pass data to renderer process in `packages/gui/src/electron/{main.tsx, preload.js}`.

Reply:
@ChiaMineJP  This has now been refactored, let me know if that is more along the lines of what we need 👍